File: OfficeManagerUI/src/app/shared/DTOs/roles-dto.ts
export interface IRolesDto {
  id:number;
  name: string;
  description: string;
  isActive: boolean;
}

export class RolesDto implements IRolesDto {
  id: number;
  name: string;
  description: string;
  isActive: boolean;

  constructor() {
    this.id = 0;
    this.name = "";
    this.description = "";
    this.isActive = true;
  }
}

File: src/OfficeManager.Application/UserRoles/Queries/RolesDto.cs
using AutoMapper;
using OfficeManager.Application.Common.Mappings;
using OfficeManager.Domain.Entities;

namespace OfficeManager.Application.ApplicationRoles.Queries
{
    public class RolesDto : IMapFrom<RoleMaster>
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }
        public bool IsActive { get; set; }
    }
}
